backend/src/main/java/se/chalmers/dat265/group1/api/digitalpart/DigitalPartController.java

```diff
@@ -23,12 +24,26 @@ public DigitalPartController(boolean debug) {
 
     @Override
     public List<DigitalPart> getAllDigitalParts() {
-        return digitalPartRepository.getObjects();
+        List<DigitalPart> resultLit = new LinkedList<DigitalPart>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)